docs/_data/blog-posts.txt

```diff
@@ -0,0 +1,9 @@
+- [Introducing JupyterHub’s Outreachy interns! — December 2022 Cohort](https://blog.jupyter.org/introducing-jupyterhubs-outreachy-interns-december-2022-cohort-23aaf4613556?source=rss----95916e268740--jupyterhub) by **Sarah Gibson**
```

Comment:
Should we gitignore this file if it's generated at build time?

Reply:
Oops yep - I wanted to fit keep the folder but forgot to gitignore everything inside.

I'll plan to fix that and then merge 👍